src/gui/windows/debug.zig

```diff
@@ -38,6 +39,19 @@ pub fn render() void {
 		y += 8;
 		draw.print("Queue size: {}", .{main.threadPool.queueSize()}, 0, y, 8, .left);
 		y += 8;
+		const perf = main.threadPool.getPerformance();
+		const values = comptime std.enums.values(TaskType);
+		inline for(values) |t| {
+			const name = switch (t) {
```

Comment:
Why not just use `@tagName`?

Reply:
To separate source code and rendering options. Suppose we added localization?

Reply:
Debug info does not need to be translated, since it shouldn't be used by the majority of players.

Reply:
I suppose that will be a secondary concern after we get any localization in place.